Sources/DocCDocumentation/DocCDocumentation.docc/customizing-your-documentation-appearance.md

```diff
@@ -0,0 +1,244 @@
+# Customizing Your Documentation Appearance
+
+Customize the look and feel of your documentation webpages.
+
+## Overview
+
+By default, rendered documentation webpages produced by DocC come with a default
+visual styling. If you wish, you may make adjustments to this styling by adding
+an optional `theme-settings.json` file to the root of your documentation catalog
+with some configuration. This file is used to customize various things like
+colors and fonts.  You can even make changes to the way that specific elements
+appear, like buttons, code listings, and asides. Additionally, some metadata for
+the website can be configured in this file, and it can also be used to opt in or
+opt out of certain default rendering behavior.
+
+If you're curious about the full capabilities of the `theme-settings.json` file,
+you can check out the latest Open API specification for the it [here][1], which
+documents all valid settings that could be present.
+
+Let's walk through some basic examples of customizing the appearance of
+documentation now.
+
+### Colors
+
+DocC-Render utilizes [CSS custom properties][2] (CSS variables) to define the
+various colors that are used throughout the HTML elements of rendered
```

Comment:
Referring to CSS variables in this paragraph and below (with `--color-documentation-intro-fill`) seems like an implementation detail, which I don't think is relevant to the audience of this documentation, which is _users_ of DocC. Is there a benefit to alluding to how DocC websites use the theme information?

Reply:
I did want to explicitly call this out to uncover the magic and make it clear what these settings are doing behind the scenes here. The main user benefit is that it makes it more clear the syntax that you would have to use if you wanted to have one var refer to another one. It is demonstrated in one of the examples, but I thought it was important to call out explicitly.